alpine-nix-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -0,0 +1,103 @@
+env PATH;
+env TZ=utc;
+
+user nginx nogroup;
+
+worker_processes auto;
+worker_priority  15;  # limit worst-case resource utilization to 25%
+
+events {
+  worker_connections 1024;
+}
+
+http {
+  # Do not leak application information
+  server_tokens off;
+  more_clear_headers Server X-Powered-By;
+
+  sendfile    on;
+  tcp_nopush  on;
+  tcp_nodelay on;
+
+  keepalive_timeout 90;
+
+  include      /nix/var/nix/profiles/default/conf/mime.types;
+  default_type application/octet-stream;
+
+  types {
+    image/icns            icns;
+    application/font-sfnt ttf otf;
+  }
+
+  source_charset utf-8;
+  charset        utf-8;
+  charset_types  text/plain text/css application/javascript application/json text/xml application/xml application/xml+rss image/svg+xml;
+
+  gzip              on;
+  gzip_vary         on;
+  gzip_static       on;
+  gzip_proxied      any;
+  gzip_disable      "msie6";
```

Comment:
Is this "feature branch" still needed? AFAIK MSIE6 is extinct enough to ignore it.